src/main/resources/liquibase/202205231334-create-billable-usage-tracking-table.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+  <changeSet id="202205231334" author="lburnett">
+    <comment>Create billable_usage_remittance tracking table</comment>
+
+    <createTable tableName="billable_usage_remittance">
+      <column name="account_number" type="VARCHAR(32)"/>
+      <column name="org_id" type="VARCHAR(32)"/>
+      <column name="product_id" type="VARCHAR(32)"/>
+      <column name="metric_id" type="VARCHAR(32)"/>
```

Comment:
For consistency, we use `VARCHAR(255)` for `uom` (which is what we plan to rename to `metric_id` at some point.

Reply:
DONE